TicketsDemo.Domain/DefaultImplementations/TicketService.cs

```diff
@@ -25,7 +26,7 @@ public class TicketService : ITicketService
             _runRepository = runRepository;
         }
 
-        public Ticket CreateTicket(int reservationId, string fName, string lName)
+        public Ticket CreateTicket(int reservationId, string fName, string lName, bool drink, bool bed)
```

Comment:
the only thing you need to avoid having two different methods with almost-the-same-code is to change the signature of this method to:

public Ticket CreateTicket(int reservationId, string fName, string lName, bool drink = false, bool bed = false)